tests/it/e2e_rpc_server.rs

```diff
@@ -0,0 +1,30 @@
+use anyhow::Ok;
+
+use crate::util::{wasm_file, Soroban};
+
+// e2e tests are ignore by default
+#[test]
+#[ignore]
+fn e2e_deploy_and_invoke_contract_against_rpc_server() -> Result<(), anyhow::Error> {
+    // This test assumes a fresh standalone network rpc server on port 8000
+
+    Soroban::new_standalone()?
+        .arg("deploy")
+        .wasm(wasm_file("test_hello_world"))?
```

Comment:
❓ Why is the `tests/fixtures/soroban_hello_world_contract.wasm` file committed as part of the PR? I'm assuming that was by accident because we don't have fixtures excluded in the gitignore.

Reply:
Yep I'll remove that.